{"name": "test_login_with_invalid_email_and_valid_password", "status": "passed", "start": 1684241535218, "stop": 1684241536710, "uuid": "ae4de312-ea18-4a86-878d-a1b5fb8ab0b2", "historyId": "9d7c15636f7b48d3beb276131c85cb96", "testCaseId": "9d7c15636f7b48d3beb276131c85cb96", "fullName": "tests.test_Login.TestLogin#test_login_with_invalid_email_and_valid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VO28CS3"}, {"name": "thread", "value": "10320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Login"}]}